README.adoc

```diff
@@ -1,85 +1,90 @@
+- *https://access.redhat.com/documentation/en/red-hat-xpaas/0/paged/red-hat-xpaas-a-mq-image/[JBoss A-MQ]*: Apache ActiveMQ-based message broker container image for receiving and storing telemetry from the Sensor via http://mqtt.org/[MQTT]
+- *https://access.redhat.com/documentation/en/red-hat-xpaas/0/paged/red-hat-xpaas-decision-server-image/chapter-1-introduction-to-the-decision-server-image/[JBoss BRMS Decision Server]*: Real-time execution of business rules for processing telemetry
+- *https://zeppelin.apache.org/[Apache Zeppelin]*: Interactive data visualization and analytics
+- *https://www.postgresql.org/[PostgreSQL]*: Provides persistence for sensor data
+- *Simulated Software Sensor*: A simple Spring Boot application that simulates temperature, vibration, and GPS sensor data
 
 == Prerequisites
 
-* Git
-* Access to an OpenShift environment
-* OpenShift Command Line Interface (CLI)
+* https://git-scm.com/[Git]
+* https://github.com/openshift/origin/releases[OpenShift command-line tools] for your platform
+* Access to an OpenShift environment.  Don't have one handy? Try https://asciinema.org/a/49402[`oc cluster up`] or the https://github.com/redhatdemocentral/cdk-install-demo[Red Hat Container Development Kit] (CDK)
 
 == Setup
-*Step 1:* Launch the OpenShift Container platform and login to OpenShift Command Line
+
+*Step 1:* Launch the OpenShift Container platform and login using the command-line tools
 
 _Note: The OpenShift environment can be enabled by CDK platform. To learn how to install CDK, check out https://github.com/redhatdemocentral/cdk-install-demo[CDK-Install Demo]_
 
-*Step 2:* Download the iot-ose project in the OpenShift instance:
+*Step 2:* Download the iot-ocp project in the OpenShift instance:
 
- [user@localhost ~]$ git clone https://github.com/sabre1041/iot-ose.git
+ $ git clone https://github.com/sabre1041/iot-ocp.git
 
+*Step 3:* Execute the init script
 
-The https://github.com/sabre1041/iot-ose/blob/master/init.sh[init.sh] script, provided with the project, automates the provisioning process for the example project.
+ $ cd iot-ocp
+ $ ./init.sh
 
-*Step 3:* Execute the init script
+The script automatically builds, deploys, and wires the necessary application components together.  Once the process is complete, you can see a graphical visualization of the application architecture using the OpenShift Web Console.
 
- [user@localhost ~]$ ./init.sh
- 
-The init script triggers synchronous builds and deployments of A-MQ, Decision Server, Fuse Integration Service, PostgreSQL, Zeppelin and Software Sensor components. OpenShift web console provides a graphical UI into the setup.
+*Step 4:* Login to the OpenShift web console using a web browser
 
-*Step 4:* Login to the OpenShift web console using a web browser (login/password - admin/admin) and select the **iot-ose** project
+**Username**: admin
+**Password**: admin
+**Project**: iot-ose
```

Comment:
We should probably update the name of the project that is created by the init script. I'll create an issue and work on the implementation. Would you be able to update to *iot-ocp* as this will be the name moving forward?